tox.ini

```diff
@@ -57,6 +57,8 @@ description = Run unit tests
 deps =
     pytest
     requests
+    pytest-mock
+    juju==3.2.0.1
```

Comment:
Shouldn't it be a separate PR to move the version, and then to add secrets?

It allows to see all potential "surprise" introduced with the version change -- independent from code changes.

Reply:
I prefer to have it in the same pr

Reply:
I agree with Judit